src/boot.ts

```diff
@@ -1,3 +1,5 @@
+require("dotenv").config()
```

Comment:
Can we use the import style here? `import 'dotenv/config'`

Reply:
Just because having code that runs before `import` statements is not really part of the JS standard I think